leetcode palak

palakbaghla opened the pull request.

palakbaghla committed:
leetcode palak

palakbaghla committed:
Leetcode java Palak

Vishnu-M commented:
Add the solution code directly to the repo i.e, inside `Leetcode/src` folder.
eg :- https://github.com/fossgect/Algorithms-Implementation/commit/e5775a9d7747622ac4815d998ae9fdbe64d3217d

palakbaghla committed:
Leetcode java Palak

palakbaghla committed:
Leetcode java Palak

abh006 commented:
Add the code files like `program.c` or `program.cpp` directly into this repo. Adding tables with links to the code is not valid. And hope you would make it in respective folders so that it becomes more readable and organized :)

palakbaghla commented:
I did that only I already changed it previously. I added java code
Thanks and Regards
Palak Baghla
(8860455314)



On Sun, Nov 3, 2019 at 9:25 PM Abhinav K M <notifications@github.com> wrote:

> Add the code files like program.c or program.cpp directly into this repo.
> Adding tables with links to the code is not valid. And hope you would make
> it in respective folders so that it becomes more readable and organized :)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossgect/Algorithms-Implementation/pull/11?email_source=notifications&email_token=AHNYCBPACCA7ZICFF2Y6EJDQR3X6ZA5CNFSM4JFYNUR2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEC5WDCI#issuecomment-549151113>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHNYCBJZKADAHMANZN46HV3QR3X6ZANCNFSM4JFYNURQ>
> .
>

abh006 commented:
The link to the code cannot be accepted. You should add code files to this. I adding files and then raising another pull request.

palakbaghla commented:
I added in the repository itself and even changed the link in the readme
file

Thanks and Regards
Palak Baghla
(8860455314)



On Sun, Nov 3, 2019 at 10:35 PM Abhinav K M <notifications@github.com>
wrote:

> The link to the code cannot be accepted. You should add code files to
> this. I adding files and then raising another pull request.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossgect/Algorithms-Implementation/pull/11?email_source=notifications&email_token=AHNYCBOQC3QW3RIJG5E5LWTQR4AGXA5CNFSM4JFYNUR2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEC5XXYY#issuecomment-549157859>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHNYCBP6PI6HVHOSBBNPUIDQR4AGXANCNFSM4JFYNURQ>
> .
>

palakbaghla commented:
I am not able to get what you are saying. Can you please elaborate?
Thanks and Regards
Palak Baghla
(8860455314)



On Mon, Nov 4, 2019 at 12:06 AM PALAK BAGHLA <notifications@github.com>
wrote:

> I added in the repository itself and even changed the link in the readme
> file
>
> Thanks and Regards
> Palak Baghla
> (8860455314)
>
>
>
> On Sun, Nov 3, 2019 at 10:35 PM Abhinav K M <notifications@github.com>
> wrote:
>
> > The link to the code cannot be accepted. You should add code files to
> > this. I adding files and then raising another pull request.
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/fossgect/Algorithms-Implementation/pull/11?email_source=notifications&email_token=AHNYCBOQC3QW3RIJG5E5LWTQR4AGXA5CNFSM4JFYNUR2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEC5XXYY#issuecomment-549157859
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AHNYCBP6PI6HVHOSBBNPUIDQR4AGXANCNFSM4JFYNURQ
> >
> > .
> >
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossgect/Algorithms-Implementation/pull/11?email_source=notifications&email_token=AHNYCBP4JPM74PNCSO2QRVDQR4K3ZA5CNFSM4JFYNUR2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEC5ZU7Y#issuecomment-549165695>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHNYCBKDOX3BMQYWHQ6J4OLQR4K3ZANCNFSM4JFYNURQ>
> .
>
